src/js/common/schemaform/review/PhoneNumberWidget.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+// When coming from a PhoneNumberWidget (not review), value will be only numbers
+export default function PhoneNumberWidget({ value }) {
+  let formatted = value;
+  if (value && value.length === 10) {
+    formatted = `(${value.substr(0, 3)}) ${value.substr(3, 3)}-${value.substr(6)}`;
+  }
```

Comment:
I'd like to pull this out into a common module to be used for any formatted string. Perhaps also supply a `transform` function to modify the value? But then how would we use that (easily) in the schema?